wallet/walletdb/interface.go

```diff
@@ -144,6 +146,11 @@ type ReadCursor interface {
 	// not exist, the cursor is moved to the next key after seek.  Returns
 	// the new pair.
 	Seek(seek []byte) (key, value []byte)
+
+	// Close would close the cursor. It is important to close the cursor
```

Comment:
Instead of saying "it is important to..." just document how it must be used:

> Close closes the cursor.  Cursors must be closed before opening a new cursor and before finishing a transaction.